src/components/button/button.hbs

```diff
@@ -0,0 +1,8 @@
+<button class="bx--btn bx--btn--{{variant}}{{#if small}} bx--btn--sm{{/if}}" type="button">Button</button>
```

Comment:
Could the variants (like small/big) just be separate hbs files?

I think Watson Health does `component--variant.hbs` as the pattern

Reply:
@joshblack Yes, that’s seen in tables in this PR

Reply:
BTW given you brought up separate files for variants - Wondering what you imagined using templates for? Text node values? Attributes? Repeat?